mvcc/kvstore_test.go

```diff
@@ -522,7 +522,7 @@ func TestHashKVWhenCompacting(t *testing.T) {
 	s := NewStore(b, &lease.FakeLessor{}, nil)
 	defer os.Remove(tmpPath)
 
-	rev := 1000
+	rev := 10000
```

Comment:
on my machine 100000 triggers everytime. I lower it to 10000 since ci machine might be slow which could trigger timeout.